dev/com.ibm.ws.springboot.support_fat/fat/src/com/ibm/ws/springboot/support/fat/CommonWebServerTests15Servlet40.java

```diff
@@ -26,18 +26,26 @@
 @RunWith(FATRunner.class)
 @Mode(FULL)
 public class CommonWebServerTests15Servlet40 extends CommonWebServerTests {
-    @AfterClass
-    public static void stopTestServer() throws Exception {
+
+    @After
+    public void stopTestServer() throws Exception {
+        String methodName = testName.getMethodName();
+
         if (!javaVersion.startsWith("1.")) {
-            server.stopServer("CWWKC0265W");
+            if (methodName != null && methodName.contains(DEFAULT_HOST_WITH_APP_PORT)) {
+                super.stopServer(true, "CWWKC0265W", "CWWKT0015W");
+            } else {
+                super.stopServer(true, "CWWKC0265W");
+            }
+        } else {
+            if (methodName != null && methodName.contains(DEFAULT_HOST_WITH_APP_PORT)) {
+                super.stopServer(true, "CWWKT0015W");
+            } else {
+                super.stopServer();
+            }
```

Comment:
Can we factor this into creating a `List<String> expectedErrors` and then doing `server.stop(true, expectedErrors.toArray())`?

Reply:
keeping existing to avoid more code.